utils/src/main/scala/cromwell/pipeline/utils/auth/AuthContent.scala

```diff
@@ -1,4 +1,4 @@
-package cromwell.pipeline.datastorage.utils.auth
+package cromwell.pipeline.utils.auth
```

Comment:
I don't think it should be in utils module. Probably in the same place where `SecurityDirective`.

Reply:
moved

Reply:
moved it into new module